pkgs/by-name/im/immich-go/package.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, buildGoModule, fetchFromGitHub, nix-update-script, testers, immich-go }:
+buildGoModule rec {
+  pname = "immich-go";
+  version = "0.13.2";
+
+  src = fetchFromGitHub {
+    owner = "simulot";
+    repo = pname;
```

Comment:
use of pname is considered an anti-pattern https://github.com/NixOS/nixpkgs/issues/277994